rules/scala.bzl

```diff
@@ -484,65 +488,79 @@ Generates Scaladocs.
 
 configure_bootstrap_scala = rule(
     attrs = {
-        "version": attr.string(mandatory = True),
         "compiler_classpath": attr.label_list(
             mandatory = True,
             providers = [JavaInfo],
         ),
-        "runtime_classpath": attr.label_list(
-            mandatory = True,
-            providers = [JavaInfo],
-        ),
         "global_plugins": attr.label_list(
             doc = "Scalac plugins that will always be enabled.",
             providers = [JavaInfo],
         ),
         "global_scalacopts": attr.string_list(
             doc = "Scalac options that will always be enabled.",
         ),
+        "runtime_classpath": attr.label_list(
+            mandatory = True,
+            providers = [JavaInfo],
+        ),
+        "semanticdb_bundle": attr.bool(
+            default = True,
+            doc = "Whether to bundle SemanticDB files in the resulting JAR. Note that in Scala 2, this requires the SemanticDB compiler plugin.",
```

Comment:
Should we provide some default for Scala 2? Is that even possible?

Reply:
Would we even want to do that if we could?

Reply:
What do you mean by this? The default is set to `true`, which matches the default for the compiler plugin (bundling SemanticDB files in the JAR).